BUILD.gn

```diff
@@ -156,18 +159,49 @@ asar("js2asar") {
       "lib/browser/api/views/text-field.js",
     ]
   }
+
+  outputs = [
+    "$target_gen_electron_js/{{source}}",
+  ]
+}
+
+asar("electron.asar") {
+  deps = [
+    ":copy_lib_js",
+  ]
+
+  root = "$target_gen_electron_js/electron/lib"
```

Comment:
@nornagon It seems quite complex to strip out those `electron`'s with `rebase_path` so I just owned them 😆 The `tsc` stuff will strip a few of them anyway so that's a win 👍

Reply:
`gen/electron` is defined by` target_gen_dir`
the "electron" in `js/electron` is required because of the `copy` using `{{source}}` (which starts with `electron`.  It's accept the weird path or do a bunch of path extraction and `rebase_path` calls.

I'd personally rather have the annoying but explainable paths than have to strip apart a deep chain of `rebase_path` and `get_path_info`